packages/melody-compiler/__tests__/__snapshots__/CompilerSpec.js.snap

```diff
@@ -2235,6 +2228,80 @@ http://twig.sensiolabs.org/doc/filters/index.html"
 
 exports[`Compiler should fail transforming wrong import.template 1`] = `"Import must be used in macro, block or template"`;
 
+exports[`Compiler should transform templates with warnings mount without as.template 1`] = `
+"
+import Footwig from \\"foo.twig\\";
+import { component } from \\"melody-idom\\";
+import Component from \\"./component\\";
+export const _template = {};
+
+_template.render = function (_context) {
+  component(Component, \\"bg/=vtO\\");
+  component(Footwig, \\"%XboJwd\\", {
+    foo: \\"bar\\"
+  });
+  component(_context.Foo, \\"GwcMyf%\\");
+  component(_context.Foo, \\"$~m0VFY\\", {
+    foo: \\"bar\\"
+  });
+};
+
+if (process.env.NODE_ENV !== \\"production\\") {
+  _template.displayName = \\"MountWithoutAs\\";
+}
+
+export default function MountWithoutAs(props) {
+  return _template.render(props);
+}
+"
+`;
+
+exports[`Compiler should transform templates with warnings mount without as.template 2`] = `
+[MockFunction] {
+  "calls": Array [
+    Array [
+      "Missing 'as' key with mount.
+> 1 | {% mount './component' %}
+    |    ^
+  2 | {% mount 'foo.twig' with { foo: 'bar' } %}
+  3 | {% mount Foo %}
+  4 | {% mount Foo with { foo: 'bar' } %}
+
+There should be a unique key passed with \\"as\\" keyword. e.g: {% mount './component' as 'comp' %}",
```

Comment:
> When mounting a component please make sure to always specify a `key`, i.e. `{% mount './component' as 'comp' %}

Or something like that might be easier to understand. Not sure we should introduce the concept of "keywords".
I'm also not sure about "unique" since it only has to be unique within the current node, not even the current file. And "unique" could really be understood in many ways, including "unique within the entire project". So either we include it but in a more precise way, or we leave it off. Do we have a documentation page which explains this hole concept in sufficient detail and which we can link to from here?